lms/djangoapps/instructor/enrollment.py

```diff
@@ -244,6 +245,15 @@ def reset_student_attempts(course_id, student, module_state_key, requesting_user
                     requesting_user_id=requesting_user_id
                 )
                 submission_cleared = True
+                SCORE_CHANGED.send(
```

Comment:
Upon further looking at the code I think my .NET is showing; `block` should be accessible anywhere after it is assigned right?